lib/Swagger/src/Discussion/Api/PermalinksApi.php

```diff
@@ -205,7 +192,8 @@ public function getThreadWithHttpInfo($site_id, $post_id)
                 $queryParams,
                 $httpBody,
                 $headerParams,
-                '\Swagger\Client\Discussion\Models\ForumThreadHalResponse'
+                '\Swagger\Client\Discussion\Models\ForumThreadHalResponse',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\Swagger\Client\Discussion\Models\ForumThreadHalResponse" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)